xAPI.md

```diff
@@ -2483,8 +2483,18 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
+Metadata is additional information about the resource.  It enables decision making, search, and discoverability within 
+systems.  In xAPI, metadata can be utilized in a variety of locations.  The most common is within <a name="actdef"> Activity Definitions </a>, 
+where the metadata describes the Activity. Most types of metadata have the opportunity to have Extensions.  
+Other uses include Verbs, Activity types, and attachments.
+
 ### 3.1 IRI Requirements
 
+Up-to-date information is the most effective. xAPI metadata is no exception, which is why xAPI uses IRIs for identifiers.
+Using IRIs ensures uniqueness and promotes resolvability is the IRI.  The LRS and Activity each have responsibilities 
+in regard to each IRI as outline below.  Activity Definitions have additional rules which can be found in <a name="actdef"> 
+this section</a>.
+
 ###### Metadata Requirements
```

Comment:
Yeah something went wrong here.  Let me figure out what I was trying to do.  I wanted to reference this section above, but wanted generic requirements in 3.1.